brage-import/src/main/java/no/sikt/nva/brage/migration/mapper/BrageNvaMapper.java

```diff
@@ -217,9 +218,21 @@ private static List<AssociatedArtifact> convertFilesToAssociatedArtifact(List<Co
     private static AssociatedArtifact generateFile(ContentFile file, Record brageRecord) {
         var legalNote = extractLegalNote(brageRecord);
         var embargoDate = defineEmbargoDate(legalNote, file);
-        return BundleType.ORIGINAL.equals(file.getBundleType())
-                   ? createPublishedFile(file, brageRecord, embargoDate, legalNote)
-                   : createAdministrativeAgreement(file, brageRecord);
+        return switch (file.getBundleType()) {
+            case BundleType.ORIGINAL -> createPublishedFile(file, brageRecord, embargoDate, legalNote);
+            case BundleType.LICENSE -> createAdministrativeAgreement(file, brageRecord);
+            case BundleType.IGNORED -> createAdministrativeAgreementForDublinCore(file, brageRecord);
+            default -> new NullAssociatedArtifact();
+        };
+    }
+
+    private static AssociatedArtifact createAdministrativeAgreementForDublinCore(ContentFile file, Record record) {
```

Comment:
I am a bit reluctant to store this metadata file as an admistrative agreement. I think this is an old version of the metadata, and should be stored as such. Administrative agreement is another thing. And storing the old version of the metadata as an associated artefact is also weird at best.